spec/features/staff/organisation_show_page_spec.rb

```diff
@@ -122,10 +131,19 @@
       expect(page.find("table.projects  tbody tr:first-child")[:id]).to have_content(yet_another_project.id)
       expect(page.find("table.projects  tbody tr:last-child")[:id]).to have_content(project.id)
     end
+
     scenario "they do not see projects that they are not the reporting organisation of" do
       expect(page).not_to have_content another_project.identifier
     end
 
+    scenario "they see a list of all their third-party projects" do
```

Comment:
Do you think we should test they do not see third-party projects that are not theirs (like we do for projects) or are we happy that is covered elsewhere?

Reply:
Can't hurt to add another test 👍